UnitsNet/CustomCode/Wrappers/Pressure.Wrapper.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnitsNet.Units;
+using UnitsNet.CustomCode.Units;
+
+namespace UnitsNet.CustomCode.Wrappers
+{
+    /// <summary>
+    /// From gratestas in #422
+    /// Pressure is a state function, for its measurement depends on environmental factors such as ambient pressure, elevation above sea level and local weather conditions. For two persons located in the different environment to speak of the same pressure, one must relate it to a reference. There are three basis reference: absolute, gauge, vacuum
+    ///    Absolute is zero-referenced to the total vacuum.
+    ///    Gauge refers to a level of the local atmospheric pressure.
+    ///    Vacuum is the negative of the gauge.
+    ///Therefore, to obtain consistent and qualitative data, the reference measurement is crucial.
+    /// </summary>
+    public struct ReferencePressure
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="pressure"></param>
```

Comment:
I would add a copy of the summaries for each property to the parameters here for convenience.

Reply:
To confirm, you are suggesting a copy of the summary of pressure and reference?